tasks.py

```diff
@@ -457,23 +456,30 @@ def update(_c, debug=False):
 
     for package in constants.APT_PACKAGES:
         version = utils.get_latest_release_from_apt(package=package)
-        config.update_config_file(thing=package, version=version)
+        config.update_config_file(package=package, version=version)
 
     for repo in constants.GITHUB_REPOS_RELEASES:
         version = utils.get_latest_release_from_github(repo=repo)
-        config.update_config_file(thing=repo, version=version)
+        config.update_config_file(package=repo, version=version)
 
     for repo in constants.GITHUB_REPOS_TAGS:
         version = utils.get_latest_tag_from_github(repo=repo)
-        config.update_config_file(thing=repo, version=version)
+        config.update_config_file(package=repo, version=version)
 
     for project in constants.HASHICORP_PROJECTS:
         version = utils.get_latest_release_from_hashicorp(project=project)
-        config.update_config_file(thing=project, version=version)
+        config.update_config_file(package=project, version=version)
+        if project == "terraform":
```

Comment:
Although `terraform` is the only member of `constants.HASHICORP_PROJECTS` right now, we don't want to assume that will always be the case.